### ISSUE TITLE
Running pulsar 2.3.0 locally as standalone throws an exception

### ISSUE THREAD

**Issue by gkatiforis** [apache/pulsar#3886]
**Describe the bug**
I am trying to run pulsar 2.3.0 locally as standalone but i am getting an exception upon startup. I have successfully run it with pulsar 2.2.1. This is the corresponding [stack trace](https://github.com/apache/pulsar/files/2997143/pulsar-logs.txt).

**To Reproduce**

Follow the instructions [here](https://pulsar.apache.org/docs/en/standalone/) to run pulsar 2.3.0 locally as standalone:

```
wget https://archive.apache.org/dist/pulsar/pulsar-2.3.0/apache-pulsar-2.3.0-bin.tar.gz
$ tar xvfz apache-pulsar-2.3.0-bin.tar.gz
$ cd apache-pulsar-2.3.0
$ bin/pulsar standalone
```
**Expected behavior**
Startup pulsar 2.3.0 locally as standalone successfully.

**Desktop (please complete the following information):**
 - OS: Linux

**Comment by lovelle** [comment]
Hi!

The problem here is that there are not enough threads for jetty to initiate. One way I found to reproduce this on linux box is: `taskset -c 0 ./bin/pulsar standalone`

Also, I wasn't able to reproduce this with latest changes (master version) therefore, this issue might be already solved for future release 2.4, but is probably worth the effort to found it and backport the fix to 2.3 version, idk.

Are you trying to startup pulsar on a very limited machine or container (only 1 cpu available)?

**Comment by merlimat** [comment]
This was fixed in #3776 and it's scheduled for 2.3.1 release

**Comment by lovelle** [comment]
Oh great! I was driving crazy trying to find where the num of threads are being defined, thanks @merlimat !
